spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -33,10 +33,8 @@
       context "with one dep" do
         let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
         it "does not error" do
-          builder_response = described_class.build(submission)
-          expect(builder_response.errors).not_to be_present
```

Comment:
[DUST] You mention that we're removing these tests because they're redundant, where else are we validating that the XML is built without errors?

Reply:
We're validating that on line 12 with the `after do`

Reply:
[dust] i know this is after the fact and not important but if we revisit this for any reason it would probably be worth rewording `it "does not error"` more to what is in the expectations... `it "has not-nil salary and income"`, maybe?